src/makeRobot.js

```diff
@@ -38,7 +38,92 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    location: `${name}: x=0, y=0`,
```

Comment:
```suggestion
   get location() {}
```
location Должен быть геттером.